de.dlr.sc.virsat.model.extension.requirements.feature/.project

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>de.dlr.sc.virsat.model.extension.requirements.feature</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>

```

Comment:
I think features should only have FeatureBuilder and maven2Builder. 
Check other features please.

Reply:
Aree. Please check.